cmd/kube-apiserver/app/server.go

```diff
@@ -94,6 +94,7 @@ type APIServer struct {
 	ClusterName                string
 	EnableProfiling            bool
 	MaxRequestsInFlight        int
+	MinRequestTimeout          int
```

Comment:
To me Connection* indicates something that is general across all connections while Request*, something that is handled per request. Since this is the latter can I leave the name as-is (or were you confused by timeout vs duration)?